changedetectionio/processors/text_json_diff.py

```diff
@@ -241,7 +242,7 @@ def run(self, uuid, skip_when_checksum_same=True, preferred_proxy=None):
                 # FYI, There is no intersection between is_html, is_json, is_binary, is_source.
                 # If child class(driver) of class Fetcher provides binary data, use it.
                 # This is for non filter.
-                if is_html and hasattr(fetcher, 'raw_content'):
+                if is_html and hasattr(fetcher, 'raw_content') and not is_pdf:
```

Comment:
As you can see, without such an `is_pdf`, this PR can break implicit sequences(cases).